src/screens/WhotoFollow/elements/ItemUser.js

```diff
@@ -41,7 +41,7 @@ const ItemUser = ({photo, username, bio, followed, onPress, userid, karmaScore})
             <IconCheck
               width={dimen.normalizeDimen(32)}
               height={dimen.normalizeDimen(32)}
-              fill={colors.signed_primary}
+              fill={COLORS.signed_primary}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fill color for the `IconCheck` component has been updated to use the new `COLORS` object. This change is also in line with the PR's objective.
```diff
- fill={colors.signed_primary}
+ fill={COLORS.signed_primary}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->